### PR TITLE
feat: add doc for pipeline

### PR DESCRIPTION
**Pull request by chengjoey**
1. add tip for pipeline execute permission
2. add doc for commands shell

### CODE REVIEW

**Comment by chengjoey** [comment]
#https://github.com/erda-project/erda/pull/5359

File: docs/next/manual/dop/guides/cicd-pipeline/pipeline-execution.md [hunk 1]
```diff
@@ -6,6 +6,11 @@
 ![](http://terminus-paas.oss-cn-hangzhou.aliyuncs.com/paas-doc/2022/02/24/1f297ff6-181e-4a2c-8e4c-326172a71527.png)
 点击右上角 **新建流水线**，将生成一条全新的流水线，再点击执行按钮运行流水线。
 
+::: tip 提示
+普通流水线中所有节点的执行权限全部根据当前执行者的权限决定。
```
Comment (sfwn) [inline docs/next/manual/dop/guides/cicd-pipeline/pipeline-execution.md:10]: what is '普通'

**Review by sfwn — CHANGES_REQUESTED** [review]

File: docs/next/manual/dop/guides/cicd-pipeline/pipeline-execution.md (hunk 1 above)
Comment (chengjoey) [reply docs/next/manual/dop/guides/cicd-pipeline/pipeline-execution.md]: adjusted, `普通` -> `手动触发`

**Review by sfwn — APPROVED** [review]

**Pull request merged**